physical/zookeeper.go

```diff
@@ -317,7 +317,7 @@ func (i *ZookeeperHALock) Lock(stopCh <-chan struct{}) (<-chan struct{}, error)
 func (i *ZookeeperHALock) attemptLock(lockpath string, didLock chan struct{}, failLock chan error, releaseCh chan bool) {
 	// Wait to acquire the lock in ZK
 	lock := zk.NewLock(i.in.client, lockpath, i.in.acl)
-	err := lock.Lock()
+	_, err := lock.Lock([]byte{})
```

Comment:
Yup, this is the only backwards incompatible change I can think of. I should probably add some window dressing like:

```go
var ZkNoData = []byte{0}
```